### PR TITLE
Add info popover to radio input

### PR DESCRIPTION
**Pull request by guerrap**

### CODE REVIEW

**Comment by Diogomartf** [comment]
where are we using radio buttons so I can check?

**Comment by luzzif** [comment]
> where are we using radio buttons so I can check?

They're getting used in the Reality.eth oracle template, more specifically here: https://github.com/carrot-kpi/reality-eth-v3-oracle-template/blob/develop/packages/frontend/src/page/components/answer-form/index.tsx#L300

File: packages/ui/src/components/data-display/timer/index.tsx [hunk 1]
```diff
@@ -53,7 +53,9 @@ export const Timer = ({
                     className={iconStyles({ className: className?.icon })}
                 />
             )}
-            {duration.format("DD[D] HH[H] mm[M] ss[S]")}
+            {duration.asSeconds() >= 0
+                ? duration.format("DD[D] HH[H] mm[M] ss[S]")
+                : "00D 00H 00M 00S"}
```
Comment (Diogomartf) [inline packages/ui/src/components/data-display/timer/index.tsx:58]: I think we use this format in several places, maybe we should make this a variable and use it everywhere.

something like this:
`const ZERO_DATE_TIME_FORMAT=00D 00H 00M 00S`

**Comment by luzzif** [comment]
Addressed the comment and added a `seconds` prop to the timer. By default seconds are hidden and can be shown with the prop. I've also fixed the frontend's campaign cards' timer spanning 2 lines.

**Pull request merged**